core/server/common/src/main/java/alluxio/master/journal/JournalTool.java

```diff
@@ -76,16 +94,19 @@
           "The start log sequence number (inclusive). Set to 0 by default.")
       .addOption("end", true,
           "The end log sequence number (exclusive). Set to +inf by default.")
+      .addOption("inputDir", true,
+          "The input directory on-disk to read journal content from. "
+              + "(Default: Read from system configuration.)")
       .addOption("outputDir", true,
-          "The output directory to write journal content to. Default: journal_dump-${timestamp}");
+          "The output directory to write journal content to. "
+          + "(Default: journal_dump-${timestamp})");
```

Comment:
Is this relative to the directory the command is run from?

Reply:
Correct.